### PR TITLE
{bio}[intel/2017b] segemehl v0.2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8048849b19750fda4063395d84ed32ef for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2047.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7d33571958c2fdb492fade68697f1616 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
rescomp1 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/b7b11328f4dead45385e6457ee82e625 for a full test report.

File: easybuild/easyconfigs/s/segemehl/segemehl-0.2.0-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
```
Comment (verdurin) [inline easybuild/easyconfigs/s/segemehl/segemehl-0.2.0-intel-2017b.eb]: @boegel - any reason to remove Pablo's credit?

Comment (boegel) [reply easybuild/easyconfigs/s/segemehl/segemehl-0.2.0-intel-2017b.eb]: Hmm, that's only there in older `segemehl` easyconfigs, not in the `segemehl-0.2.0-foss-2016b.eb` that I used as a base.

I'll add it back, both in this easyconfig & in `segemehl-0.2.0-foss-2016b.eb`, thanks for bringing this up.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**